### ISSUE TITLE
Contribution drops but miners are online and running

### ISSUE THREAD

**Issue by brasrox** [hpool-dev/chia-miner#471]
using lastet v1.3.0-6
w10 gui version
3 miners, all online but someway today the contrinbution drops half!

tried restart, reboot, rename miner but nothing... 
324 plots and now only show 150 on online plots!

all 324 plots are visible but today they just gone on the online plot...

![image](https://user-images.githubusercontent.com/21345109/119274384-083cb300-bbe6-11eb-8f18-ff0f00fa2b38.png)
![image](https://user-images.githubusercontent.com/21345109/119274395-138fde80-bbe6-11eb-8c7b-285d52f8d3ed.png)
![image](https://user-images.githubusercontent.com/21345109/119274410-1b4f8300-bbe6-11eb-8bce-8643e8799f2d.png)

![image](https://user-images.githubusercontent.com/21345109/119274348-d3306080-bbe5-11eb-8156-c903239dc94f.png)

**Comment by brasrox** [comment]
![image](https://user-images.githubusercontent.com/21345109/119276008-b77d8800-bbee-11eb-88e4-fa6027db118e.png)

they come online now. lets see

**Comment by brasrox** [comment]
just for information, the contribution become normal today and they **added a newlabel information**  on online plots tab:

![image](https://user-images.githubusercontent.com/21345109/119392173-c5daab00-bca5-11eb-8511-b36cedb8ddeb.png)

**Comment by gaddars** [comment]
I increase my plot number on a daily basis, but my revenues are falling down. (It started to happen in last 3days)

İs it the same for you?

**Comment by brasrox** [comment]
yes, after upgrade to new version 1.4 the contribution normalized. but the revenues are falling down every day...
my guess is because the network is growing too fast! thats sad

![image](https://user-images.githubusercontent.com/21345109/119843918-5e5b7000-bede-11eb-9c3d-e506bdb6d302.png)
marked red the day when i post this issue

**Comment by cexmad** [comment]
Did you fix the contribution drop issue?

**Comment by cexmad** [comment]
> yes, after upgrade to new version 1.4 the contribution normalized. but the revenues are falling down every day...
> my guess is because the network is growing too fast! thats sad

Same issue here, already updated for 1.4.x.
